_posts/2021-04-21-unstructured-knowledge-sharing.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: epic
+title: "Unstructured Knowledge Sharing"
+date: 2021-04-21
+categories: [teams, culture, people]
+author: steve-hicks
+---
+
+- Ash wrote about _structured_ knowledge shares - /blog/2020/12/09/share-your-knowledge/
+  - but we do some good _unstructured_ knowledge sharing. Not scheduled, ad hoc.
+  - probably link somewhere to knowledge sharing as a core principle in some docs somewhere?
+- The dev-help slack channel - cc @damassi https://twitter.com/ArtsyOpenSource/status/1357819867638812672
+  - a place to ask technical questions when you're stuck
+  - we've built a large knowledge-base
+  - link to auto-solved bot
+  - sometimes it's not a question - it's something we already figured out but thought it was worth sharing (contribution to the knowledge base)
+  - anything interesting to say about using tooling you already have (slack)? 
+- Firing up ad hoc zooms - we've been doing more and more of this lately and it's what prompted me to request this post. 
+  - Sometimes they're more informative, like when @anandaroop recently spun up a zoom to walk through how he'd use observable to explore artist search index weighting.
+    - kind of like streaming but for your coworkers
+  - Sometimes they're more of a mobbing exercise, like when @admbtlr recently spun one up to work on some react-native navigation refactoring.
+  - In all cases they're a "push" instead of a "pull"
+    - radiating intent on what you plan on working on
+    - giving the rest of the team an opportunity to learn
+      - or contribute if they've got expertise
+- Pairing - this isn't an unusual practice and @yuki24 wrote it about in 2018 (/blog/2018/10/19/pair-programming/)
+  - but I think we're pretty good at it as an org and it's a big part of our unstructured knowledge sharing
+  - What _is_ different from many places is that we don't just do it when we're stuck, we do it to build features
+  - Knowlege-sharing of more than just code & product -- how do I approach a problem, what's my tooling, what are my techniques for debugging/testing/tightening my feedback loop
+  - We tend to focus on pairing within a team; could probably do more cross-team pairing
+- Sharing via PRs
+  - link to article on context in PRs
+
+
+Thought: If I expand this to include unstructured & structured things not covered by Ash's article, I can include: 
+- "office hours"
+- show & tell/lunch & learn
+- peer learning groups
```

Comment:
My initial idea for this post was to focus on _unstructured_ knowledge sharing, but I realized that if I don't limit myself to _unstructured_ and include _structured_ knowledge sharing that just wasn't covered by Ash's previous articles, I could mention a few other things we do ☝️ 

@mdole @pvinis and anyone else who might have thoughts about this -- do you think it would be worth expanding this article to include the items listed here (and possibly more)?

Reply:
@pepopowitz IMO I think it's fine if this is a shorter, more restricted post that's specific to unstructured knowledge sharing. I think either way it'll be a great post, but don't hesitate to stay laser-focused. You could even say in the intro "we also do more structured [matt note: I would actually argue that these are unstructed but _planned_, as opposed to the _unplanned_ shares you discuss in this article] sharing, like the knowledge shares Ash wrote about (link obvi), office hours, show & tell, and peer learning groups, but today, I want to tell you why I think unstructured shares are the bee's knees"